crates/webidl/src/first_pass.rs

```diff
@@ -503,6 +503,10 @@ impl<'src> FirstPass<'src, (&'src str, ApiStability)> for weedle::interface::Int
                 log::warn!("Unsupported WebIDL Setlike interface member: {:?}", self);
                 Ok(())
             }
+            InterfaceMember::AsyncIterable(_iterable) => {
```

Comment:
This is unrelated to undefined/void changes, it's just a new variant added between weedle 0.11 and weedle `master` also noticed in https://github.com/rustwasm/wasm-bindgen/pull/2373#issuecomment-762931655